performant-software/Annotation-Studio
# Allow admins to delete documents.

## What does this PR do?
1. Allows admins to delete documents on the documents index page.

## How to test?
1. Login as an admin and go to https://staging.covecollective.org/documents?docs=all
2. Try to delete a document and make sure the popup warning comes.
3. Now click ok and make sure you no longer see the document on https://staging.covecollective.org/documents?docs=all
